editor.planx.uk/src/pages/FlowEditor/components/Sidebar/index.tsx

```diff
@@ -220,12 +235,20 @@ const Sidebar: React.FC = React.memo(() => {
           </Header>
           <TabList>
             <Tabs onChange={handleChange} value={activeTab} aria-label="">
+              {hasFeatureFlag("TEMPLATES") && isTemplatedFrom && (
+                <StyledTab value="Customise" label="Customise" />
+              )}
```

Comment:
"Customise" language very much still to-review in a future design call, but safely feature-flagged for now